packages/react/src/core/FileTree.tsx

```diff
@@ -262,3 +291,7 @@ function isHiddenFile(filePath: string, fileName: string, hiddenFiles: Array<str
     return pathOrRegex.test(filePath);
   });
 }
+
+function getDepthStyle(depth: number) {
+  return { paddingLeft: `${12 + depth * NODE_PADDING_LEFT}px` };
```

Comment:
I hope someday we can make that a bit better. At a minimum using `rem` instead of `px` and then maybe using a `calc` and a CSS variable to make it possible to tweak the indentation.

Reply:
This might get fixed during https://github.com/stackblitz/tutorialkit/issues/326 automatically. But yep I agree on all points.